src/components/Snippet.tsx

```diff
@@ -1,4 +1,4 @@
-import React, { useEffect, useState, useRef, useCallback } from 'react'
```

Comment:
If we don't delete the unused values, we will get a TypeScript error when building in the production environment.